.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 
 language: node_js
 node_js:
-  - "10"
+  - lts/*
```

Comment:
this seems to end up as 10 anyway?

Reply:
yep, but so we don't have to bump it each time, once Node.js will release a new LTS version.

Reply:
ah cool